extensions/extensions-docker/aissemble-versioning/src/main/resources/docker/Dockerfile

```diff
@@ -1,33 +1,47 @@
 # Script for creating base AIOps Versioning image
-
 ARG DOCKER_BASELINE_REPO_ID
 ARG VERSION_AISSEMBLE
-FROM ${DOCKER_BASELINE_REPO_ID}boozallen/aissemble-fastapi:${VERSION_AISSEMBLE} AS builder
 
-RUN python -m venv /opt/venv
-ENV PATH="/opt/venv/bin:$PATH"
+#HABUSHU_BUILDER_STAGE - HABUSHU GENERATED CODE (DO NOT MODIFY)
+FROM python:3.11-slim AS habushu_builder
+# Poetry and supporting plugin installations
+RUN python -m ensurepip --upgrade && \
```

Comment:
S: not for adjudication here, but I wonder if we would benefit from just doing most of this in a docker image one and then using that as a builder?

Reply:
Yeah it's an interesting thought.  Because the builder image and the final image have to have the same base platform, Habushu allows you to set the base image and doesn't assume anything about that image other than `python` is resolvable on the `PATH`.  For aissemble, we could disable Habushu's dockerfile management but I think the two things you'd get (build speed and ease of maintenance) are covered by the docker layer cache and Habushu's code generation (respectively).  Could be missing a trick though.